default_config.yaml

```diff
@@ -123,6 +123,37 @@ astromatic:
   # config files are. Ignored if config_dir is not null
   config_subdir: data/astromatic_config
 
+coaddition:
+  coaddition:
+    method: zogy
+    noise_estimator: sep
+    flag_fwhm_factor: 1.0
+    alignment:
+      method: swarp
+      to_index: last
+    inpainting:
+      multi_image_method: median
+      feather_width: 2
+      rescale_method: median
+      single_image_method: biharmonic
+      ignore_flags: 0
+  extraction:
+    measure_psf: true
+    threshold: 3.0
+    method: sextractor
+  astro_cal:
```

Comment:
For the things that also have general settings (extraciton, astro cal, photo cal), it'd be good if in the config we had a way of saying "just use the general settings for coadd".  It's nice to have the ability to customize it and use something different for coadd, but it'll be a pain to have to change things in two places if you do want to use the same parameters.

Reply:
I've made the coaddition pipeline load the generic parameters first, and override them with the "coaddition" keys.